app/controllers/taxons_controller.rb

```diff
@@ -0,0 +1,37 @@
+class TaxonsController < ApplicationController
+  before_action :redirect_to_www, unless: :new_navigaton_enabled?
+
+  def show
+    render :show, locals: {
```

Comment:
I can see how this looks like a large set of variables being passed around. I don't think instance variables are a better approach, it's often confusing and if you miss setting it we get a `nil` rather than an actual error.

I'm happy to only send the `taxon` to the view and call out to the several pieces of information from within the view. What do you think?

I expanded these locals so people know what's really needed to render a show page. Wrapping it in another presenter could be less explicit I think.